src/main/java/com/smartbear/readyapi/client/execution/RecipeExecutor.java

```diff
@@ -23,13 +23,18 @@
     private HttpBasicAuth authentication;
     private final List<ExecutionListener> executionListeners = new CopyOnWriteArrayList<>();
 
```

Comment:
As a next step, I think we should have a number of static factory methods, and then deprecate the constructor:

createSecureRecipeExecutor(String host)
createSecureRecipeExecutor(String host, int port)
createNonSecureRecipeExecutor(String host)
createNonSecureRecipeExecutor(String host, int port)